src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx

```diff
@@ -252,45 +252,31 @@ Before you install the Java agent, ensure your system meets these requirements:
     Compatible:
 
     * IBM JVM versions 8 for Linux
-
     * Eclipse OpenJ9 versions 8 to 13 for Linux, Windows, and macOS
-
     * OpenJDK versions 8 to 18 for Linux, Windows, and macOS
-
     * AdoptOpenJDK versions 8 to 16 for Linux, Windows, and macOS
-
     * Temurin version 17 for Linux, Windows, and macOS
-
     * Oracle Hotspot JVM versions 8 to 16 for Linux, Solaris, Windows, and macOS
-
     * Azul Zing JVM versions 8 and 11 for Linux, Windows, and macOS
-
     * Azul Zulu JVM versions 8 to 12 for Linux, Windows, and macOS
-
-    * Amazon Corretto JVM versions 8, 11, and 17 for Linux, Windows, and macOS
-
+    * Amazon Corretto JVM versions 8, 11, and 17 for Linux, Windows, and macO
     * Alibaba Dragonwell JVM versions 8 and 11 for Linux, Windows, and macOS
 
-      (Java 1.7) Compatible only with [Java agent 6.5.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/6.5.3/newrelic-java-6.5.3.zip) \[ZIP | 16.8 MB] legacy agent:
+    (Java 1.7) Compatible only with [Java agent 6.5.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/6.5.3/newrelic-java-6.5.3.zip) \[ZIP | 16.8 MB] legacy agent:
 
     * OpenJDK and AdoptOpenJDK JVM versions 7
-
     * IBM JVM version 7
-
     * Oracle Hotspot JVM version 7 for Linux, Solaris, Windows, macOS
 
-      (Java 1.6) Compatible only with [Java agent 4.3.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/4.3.0/newrelic-java-4.3.0.zip) \[ZIP | 9.9 MB] legacy agent:
+    (Java 1.6) Compatible only with [Java agent 4.3.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/4.3.0/newrelic-java-4.3.0.zip) \[ZIP | 9.9 MB] legacy agent:
 
     * Apple Hotspot JVM version 6 for macOS
-
     * IBM JVM version 6
-
     * Oracle Hotspot JVM version 6.0 for Linux, Solaris, Windows, macOS
 
-      (Java 1.5) Compatible only with [Java agent 2.21.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/2.21.7/newrelic-java-2.21.7.zip) \[ZIP | 2.8 MB] legacy agent:
```

Comment:
Same as previous comment. Is this supposed to be a bullet or a separate section?